src/herad.cpp

```diff
@@ -86,7 +87,10 @@ bool isHSQ(uint8_t * data, int size)
 		#endif
 		return false;
 	}
-	if ( *(uint16_t *)(data + 3) != size )
+
+	const uint16_t temp_size = u16_unaligned(data + 3);
+
+	if ( temp_size != size )
```

Comment:
```suggestion
	if (temp_size != size)
```
I added surrounding spaces inside brackets before only for better readability, with these changes they are not needed anymore. :slightly_smiling_face:

Reply:
@binarymaster the really correct way of approaching this is to use something like `clang-format` to enforce a consistent formatting :wink:.